### DISCUSSION TITLE
Why are question marks in open ranges allowed to be repeated?

### DISCUSSION THREAD

**Discussion by vladdeSV** [jotaen/klog#92] [Help / Question] [answered]
One thing which I personally never understood is why one would want to repeat the `?` in open ranges.

What was the reason for allowing this?

**Comment by jotaen** [comment, marked as answer]
The idea behind it was to give people more possibilities to visually align entry summaries if they want:

```
2020-01-01
  14:00 - 14:30 Some text
  15:30 - ? More text (not aligned)

2020-01-01
  14:00 - 14:30 Some text
  15:30 - ?     More text (aligned through whitespace)

2020-01-01
  14:00 - 14:30 Some text
  15:30 - ????? More text (aligned through ?s)
```

I personally never use that notation, though 😄  But I also don’t align my entry summaries otherwise

**Reply by vladdeSV** [reply]
Thank you for your quick answer :)

**Reply by ccjmne** [reply]
I am a fanatical horizontal aligmer myself! But I throw in the towel with Klog format, specifically because we can't have this:
```
2020-02-20
   <01:00 - 02:00
    09:00 - 18:00
    23:00 - 01:00>
```
I thought for a second that was covered, as the spec mentioned *three* spaces being a legal indentation level... alas, it may not be mixed with 4-spacers. Which is reasonable.
And, of course, I'll survive, all of this really doesn't need to be aligned in the first place ;)
